modules/ROOT/pages/waf-rulesets.adoc

```diff
@@ -0,0 +1,115 @@
+= WAF Rulesets
+
+You can enable rules by threat category for requests and responses--these are called "rulesets." Custom rules or modifications to rules are not currently available. There is an option to disable individual rules within a ruleset if they are causing false positives. See xref:create-waf-policy#disable_rules[Disable Individual Rules].
```

Comment:
Remove "Custom rules or modifications to rules are not currently available.". Why point out missing functionality?

Reply:
Yes!  💯